.gitignore

```diff
@@ -5,6 +5,7 @@ lightning_logs/
 **lightning_logs/
 **/__MACOSX
 datasets/
+dist/
```

Comment:
Added this because I built lightly a couple of times and the build outputs are stored in `dist/`. We never want to commit these files to git.